invenio_theme_tugraz/views.py

```diff
@@ -63,9 +63,13 @@ def cast_to_dict(attr_dict):
 
 def index():
     """Frontpage."""
+    records = FrontpageRecordsSearch()[:5].sort("-created").execute()
+    for r in records:
+        r = UIJSONSerializer().serialize_object_to_dict(r.to_dict())
```

Comment:
it would be nice to say something about the necessity of such changes in the commit message.